druid/src/widget/switch.rs

```diff
@@ -181,6 +181,8 @@ impl Widget<bool> for Switch {
     fn update(&mut self, ctx: &mut UpdateCtx, old_data: &bool, data: &bool, _env: &Env) {
         if old_data != data {
             ctx.request_paint();
```

Comment:
`request_paint` is no longer necessary here as we call it from `request_anim_frame` now.